autosubmit/scripts/autosubmit.py

```diff
@@ -19,16 +19,18 @@
 
 """Script for handling experiment monitoring"""
 import argparse
-import os
 import sys
 from typing import Optional
-
 from contextlib import suppress
-script_dir = os.path.abspath(os.path.dirname(sys.argv[0]))
-sys.path.append(script_dir)
-sys.path.append(os.path.normpath(os.path.join(script_dir, os.pardir)))
+
 # noinspection PyUnresolvedReferences
 from log.log import Log, AutosubmitCritical, AutosubmitError  # noqa: E402
+import sys
+import os
+
+# Add the directory containing the `autosubmit` module to sys.path
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../')))
+
```

Comment:
See comment in the test file; I think this can be removed... Manipulating the PYTHONPATH/sys.path is normally fine, but you can have unexpected results depending on what you are doing and your libraries installed... so always safer to avoid it if we can...